docs/integrations/engines/bigquery.md

```diff
@@ -1,5 +1,117 @@
 # BigQuery
 
+## Introduction
+
+This guide provides step-by-step instructions on how to use SQLMesh with the BigQuery SQL engine. It assumes you have an existing BigQuery project with CLI/API access enabled and billing configured. The guide will walk you through the steps of installing SQLMesh for BigQuery locally and connecting the Quickstart project tables.
+
+## Prerequisites
+
+- Existing BigQuery project
+- [CLI/API access enabled](https://cloud.google.com/sdk/gcloud/reference/init)
+- [Billing configured](https://console.cloud.google.com/billing) (i.e. its not a sandbox project)
+- User account with permissions to execute commands against the project
+
+## Installation
+
+### Follow the Quickstart guide
+
+Up to the installing the [sqlmesh core step](https://sqlmesh.readthedocs.io/en/stable/installation/#install-sqlmesh-core), this is where we will deviate slightly 
+
+Instead of installing just SQLMesh core we will specify the BigQuery engine
+```
+pip install "sqlmesh[bigquery]"
+```
+### Install Google Cloud SDK 
+
+Follow these steps to install the Google Cloud SDK to the new environment that you just made:
+
+- Download the appropriate package for your system from [Google Cloud installation guide](https://cloud.google.com/sdk/docs/install)
+- Unpack the downloaded file:
+    
+    ```bash
+    tar -xzvf google-cloud-cli-darwin-{SYSTEM_SPECIFIC_INFO}.tar.gz
+    ```
+    
+- Run the installation script:
+    
+    ```bash
+    ./google-cloud-sdk/install.sh
+    ```
+    
+- Reload your shell profile (e.g., for zsh):
+```
+source $HOME/.zshrc
+```
+## Configuration
+
+### Configure SQLMesh for BigQuery
+
+Add the following to your SQLMesh config.yaml file:
+
+```yaml
+bigquery:
+  connection:
+    type: bigquery
+    project: <your_project_id> 
+
+default_gateway: bigquery
+```
+
+In bigquery, navigate to the dashboard and select project your going to work with to get your project ID and copy that into the SQLMesh config.yaml
+
+![BigQuery Dashboard](./bigquery/bigquery-1.png)
+
+- Docs-Demo is the one we will use
+    - the project ID for this example is: `healthy-life-440919-s0`
+
+![BigQuery Dashboard: selecting your project](./bigquery/bigquery-2.png)
```

Comment:
It wasn't clear to me how to get to this menu - could we add another screenshot showing how you pull this up?

Reply:
I added context to this, as we discussed. 

The arrow opens the pop-up menu (it's not a dropdown). Added this context and re-ordered the image and text to ake this clearer